src/plugins/cosmos/components/modals/Staking/Staking.tsx

```diff
@@ -86,11 +86,13 @@ const StakingModalContent = ({ assetId }: StakingModalProps) => {
     close()
   }
 
-  const asset = (_ => ({
-    name: 'Osmosis',
-    symbol: 'OSMO',
-    caip19: assetId
-  }))(assetId) as Asset
+  const asset = useAppSelector(state => selectAssetByCAIP19(state, assetId))
+  const accountSpecifiersForChainId = useAppSelector(state =>
+    selectPubkeyishByChainId(state, asset?.caip2)
+  )
+  const accountSpecifier = accountSpecifiersForChainId?.[0]
```

Comment:
I dont think we should be calling this accountSpecifier here or in any of the selectors. Its not the same accountSpecifier we use other places and makes it confusing

Reply:
Let's rename it after release after we agree on new terminology and standard format? That would require renaming it in many places, in code that's in develop as well as code that's on various PRs currently so it's a bit tricky to tackle it all at once while we have PRs opened

Reply:
totally agree. I just wanted to make a note of it here so we dont forget